### PR TITLE
travis: add riot testbuild

### PR DESCRIPTION
**Pull request by kaspar030**
This PR tries to add a build of hello-world for all installed toolchains.

The idea is to get a basic notion of whether the container is able to build RIOT.
I have randomly selected one board for each installed toolchain.

### CODE REVIEW

**Comment by kaspar030** [comment]
Ok, now I got it right. ;) @miri64 wanna take a look?

File: .travis.yml [hunk 1]
```diff
@@ -7,3 +7,9 @@ services:
 script:
   - docker build -t riotdocker .
   - docker image ls riotdocker:latest
+  - git clone --depth 1 https://github.com/RIOT-OS/RIOT -b 2019.01
+  - sed -i '35i\ \ BOARDS \\' RIOT/makefiles/docker.inc.mk
+  - DOCKER_IMAGE=riotdocker:latest
+    BUILD_IN_DOCKER=1
+    BOARDS="arduino-uno esp32-wroom-32 hifive1 msb-430h native pic32-wifire samr21-xpro"
```
Comment (miri64) [inline .travis.yml]: Does it make sense to add also `native:llvm` and `samr21-xpro:llvm`?

File: .travis.yml [hunk 2]
```diff
@@ -7,3 +7,9 @@ services:
 script:
   - docker build -t riotdocker .
   - docker image ls riotdocker:latest
+  - git clone --depth 1 https://github.com/RIOT-OS/RIOT -b 2019.01
```
Comment (miri64) [inline .travis.yml]: 2019.04?

File: .travis.yml [hunk 3]
```diff
@@ -7,3 +7,9 @@ services:
...
+  - sed -i '35i\ \ BOARDS \\' RIOT/makefiles/docker.inc.mk
```
Comment (miri64) [inline .travis.yml]: What is this?

File: .travis.yml (hunk 1 above)
Comment (kaspar030) [reply .travis.yml]: done

File: .travis.yml (hunk 2 above)
Comment (kaspar030) [reply .travis.yml]: yup

File: .travis.yml (hunk 3 above)
Comment (kaspar030) [reply .travis.yml]: added a hint. without the fix, BOARDS won't propagated into the contianer, making this test build *everything*. We know how travis likes our full builds...

**Comment by kaspar030** [comment]
@miri64 I think I got them all

**Comment by miri64** [comment]
Please squash.

**Comment by miri64** [comment]
Please prefix commit message ;-).

**Comment by kaspar030** [comment]
> Please prefix commit message ;-).

yup, did. Thanks for reviewing!

**Comment by miri64** [comment]
Re-ACK

File: .travis.yml (hunk 3 above)
Comment (miri64) [reply .travis.yml]: BTW. The first two steps would have been better placed in [`before_script:`](https://docs.travis-ci.com/user/customizing-the-build), but oh well...

**Pull request merged**